code/game/machinery/doors/airlock.dm

```diff
@@ -35,8 +35,15 @@
 	var/secured_wires = 0
 	var/datum/wires/airlock/wires = null
 	var/obj/item/device/magnetic_lock/bracer = null
+
 	var/open_sound_powered = 'sound/machines/airlock.ogg'
+	var/close_sound_powered = 'sound/machines/AirlockClose.ogg'

```

Comment:
I was going to pester you about making these into statics. Then I realized that being able to change this per airlock is actually good. So nvm.